src/components/visualizations/chart/test/loadEvent.spec.ts

```diff
@@ -0,0 +1,84 @@
+// (C) 2007-2018 GoodData Corporation
+import handleChartLoad from '../events/load';
+
+const buildChart = (
+    axisGroupAddClass: () => {},
+    labelGroupAddClass: () => {},
+    opposite: boolean
+) => ({
+    yAxis: [
+        {
+            axisGroup: {
+                addClass: axisGroupAddClass
+            },
+            labelGroup: {
+                addClass: labelGroupAddClass
+            },
+            opposite
+        }
+    ]
+});
+
+const addSecondaryAxisToChart = (
+    chart: any
+) => {
+    const { yAxis: [ primaryAxis ] } = chart;
+    return {
+        yAxis: [
+            primaryAxis,
+            {
+                ...primaryAxis,
+                opposite: !primaryAxis.opposite
+            }
+        ]
+    };
+};
+
+const executeTest = (opposite: boolean, isDualAxesChart: boolean) => {
```

Comment:
Watch the spelling of `isDualAxisChart`.

Reply:
`opposite` argument shouldn't be needed - `isDualAxisChart` argument covers needed cases.